acceptance-test/user-pathways/happy-path/happy-path.test.js

```diff
@@ -17,139 +18,139 @@ const {
     WHO_EXPLOITED_PV,
     ANY_OTHER_PVS_NO_OPTION,
     PV_HAS_CRIME_REFERENCE_NUMBER_YES_OPTION,
+    REFER_CASE_TO_NRM_YES_OPTION,
 } = selectors;
 
 const APP_CONTAINER_PORT = process.env.PORT || 8081;
 let APP_CONTAINER_HOST;
 
+let browser;
 let page;
-let url;
 
-/**
- * Select the continue button a number of times specified within the
- * supplied parameters
- *
- * @param {number} loopCount - the number of time the loop should run
- *
- * @returns {void}
- */
-const clickContinueButton = async(loopCount) => {
-    for (let i = 0; i < loopCount; i++) {
-            await page.waitForSelector(CONTINUE_BUTTON);
-            await page.click(CONTINUE_BUTTON);
-    }
-};
-
-describe('Critical user path(s)', () => {
+describe.only('User path(s)', () => {
```

Comment:
why `.only`?

Reply:
This will run only this test suite. This is an alternative to commenting out the upload file tests. @sulthan-ahmed